src/Microsoft.IdentityModel.Logging/LogMessages.cs

```diff
@@ -40,6 +40,7 @@ internal static class LogMessages
         internal const string MIML10000 = "MIML10000: eventData.Payload is null or empty. Not logging any messages.";
         internal const string MIML10001 = "MIML10001: Cannot create the fileStream or StreamWriter to write logs. See inner exception.";
         internal const string MIML10002 = "MIML10002: Unknown log level: {0}.";
+        internal const string MIML10003 = "MIML10003: Can not add/remove/update default parameters: {0}. They are added by default.";
```

Comment:
Maybe rephrase this to say something along the lines of "they are added by default and cannot be modified in anyway"?